telnyx/api_resources/abstract/nested_resource_class_methods.py

```diff
@@ -4,9 +4,17 @@
 from telnyx.six.moves.urllib.parse import quote_plus
 
 
-def nested_resource_class_methods(resource, path=None, operations=None):
+def nested_resource_class_methods(
+    resource, path=None, operations=None, pluralize_path=True
+):
+    def make_path(v):
```

Comment:
Will this work for an resource ended with `y`?
E.g: `emergency -> emergencys` ❌  ?

Reply:
No. The `nested_resource_class_methods` function didn't take much into account when it came to assuming an `s` suffix for method names.

I'll keep the function as it is, for now, and tackle this issues as they arise.